.github/workflows/test.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           python-version: 3.9
       - name: Install dependencies
         run: |
-          pip install -r requirements.txt
+          pip install -e .
```

Comment:
I suggest we keep the requirements file and pin psutil there, for now. The purpose of that file is to have reproducible builds in CI and fixed dependencies for development.

Reply:
I agree. Only removed it as it didn't pin a specific version for `psutil` either.
Restored it with my last commit.